tests/test_hardware_parity_training_views.py

```
@@ -0,0 +1,108 @@
+                        "validator-derived hardware projection" in error
+                        for error in errors
+                    ),
+                    errors,
+                )
+
+    def test_swapped_view_ids_are_rejected_against_their_source_records(self):
+        records = copy.deepcopy(_fixture_records()[:2])
+        views = [hp.training_view(record) for record in records]
+        views[0]["id"], views[1]["id"] = views[1]["id"], views[0]["id"]
+        errors = []
+        for index, (record, view) in enumerate(zip(records, views), 1):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)